gr/lessons/basics/documentation.md

```diff
@@ -103,28 +116,33 @@ iex> h Greeter.hello
 
                 def hello(name)
 
-Τυπώνει ένα μήνυμα χαιρετισμού
+Prints a hello message
 
-Παράμετροι
+Parameters
 
-  • name: Αλφαριθμητικό που αναπαριστά το όνομα του ατόμου.
+  • name: String that represents the name of the person.
 
-Παραδείγματα
+Examples
 
     iex> Greeter.hello("Sean")
-    "Γειά σου, Sean"
+    "Hello, Sean"
 
     iex> Greeter.hello("pete")
-    "Γειά σου, pete"
+    "Hello, pete"
 
 iex>
 ```
 
 Παρατηρείτε πως μπορείτε να χρησιμοποιήσετε σημάνσεις μέσα στην τεκμηρίωση και το τερματικό θα τις εμφανίσει;  Εκτός από το να είναι πολύ συναρπαστικές και μια καινοτομική προσθήκη στο οικοσύστημα της Elixir, γίνεται πολύ πιο σημαντική όταν ασχοληθούμε με το ExDoc για να παράγουμε τεκμηρίωση σε HTML δυναμικά.
 
+**Σημείωση**: το σχόλιο '@spec' χρησιμοποιείται για την στατική ανάλυση κώδικα
+Για να μάθετε περισσότερα για αυτό, επισκεφτείτε το μάθημα [Specifications and types](../../advanced/typespec).
```

Comment:
Please translate the [Specifications and types] to the Greek translation of the lesson ([Προδιαγραφές και τύποι])